plugins/orchestrator-backend/package.json

```diff
@@ -53,7 +53,8 @@
     "prepack": "backstage-cli package prepack",
     "postpack": "backstage-cli package postpack",
     "postversion": "yarn run export-dynamic",
-    "export-dynamic": "janus-cli package export-dynamic-plugin"
+    "export-dynamic": "janus-cli package export-dynamic-plugin",
+    "import-editor-envelope": "scripts/import-editor-envelope.sh"
```

Comment:
First of all, let me clarify, this script is not meant to be a permanent solution. I rather make it part of the swf-editor-envelope package (please continue reading, the answer's below).

> Shouldn't we use this script to embed the envelope inside the distribution folder during export-dynamic so that it gets available within the published npm package?

No. `export-dynamic` has nothing to do with the way the envelope files are served. The envelope files are not served by the static file server that's spun up by the the dynamic-frontend-plugins plugin. Hope this addresses the question correctly, if not let me know.

> How is the community supposed to use the feature if they don't have access to the source and are only importing the npm package into the helm chart?

The way I see it, is actually the other way around. If you are in a disconnected environment, then your helm chart should also add this package as a dependency for the orchestrator-backend. What I want to do is to add an `install` script to the new swf-editor-envelope package that copies (perhapps even links) the files in its dist folder to the correct folder inside the orchestrator-backend's static folder, as per the instructions in the former package.

Reply:
Apparently `postinstall` with `yarn` [won't work](https://github.com/yarnpkg/yarn/issues/5476) :disappointed:

Reply:
I'm removing all the scripts for now, until we figure out how to "autoinstall" this package into the `orchestrator-backend`.
For testing it locally, I'm  linking the `dist` directory with this script, it needs to be executed from the root directory of the monorepo.

```sh
#!/bin/bash

set -eu

envelope_dir="$(realpath ./plugins/orchestrator-backend/static/generated/envelope)"
dist_dir="$(realpath ./plugins/orchestrator-swf-editor-envelope/dist)"

if [[ -L "$envelope_dir" ]];then 
    rm "$envelope_dir"
fi

ln -sv "$dist_dir" "$envelope_dir"
```

Reply:
Addressed in 6668713efe2fa2406dd44d62b56e3ca7411af28b